india_compliance/gst_india/overrides/sales_invoice.py

```diff
@@ -214,6 +214,7 @@ def update_dashboard_with_gst_logs(doctype, data, *log_doctypes):
         {
             "e-Waybill Log": "reference_name",
             "Integration Request": "reference_docname",
+            "GST Inward Supply": "link_name",
```

Comment:
Is this required in the Sales Invoice?

Reply:
This is a utility function,this is called in purchase invoice also.just updating non standard field.